pages/policies/new.js

```diff
@@ -69,6 +80,21 @@ const NewPolicy = () => {
     router.push(`/policies/${id}`);
   };
 
+  const onValidate = async (event) => {
+    event.preventDefault();
+
+    const response = await fetch("/api/policies/validate", {
```

Comment:
this endpoint is no longer valid

Reply:
how is this working on my local !!!

Reply:
Lol, this is actually an endpoint in the backend of the UI - if you look at the file `pages/api/policies`, you'll see the correct url

Reply:
i'm an idiot, it just looked exactly like the last endpoint and i didn't explore further